libraries/botframework-connector/src/auth/emulatorValidation.ts

```diff
@@ -95,12 +99,16 @@ export module EmulatorValidation {
     export async function authenticateEmulatorToken(
         authHeader: string,
         credentials: ICredentialProvider,
+        channelService: string,
```

Comment:
Same comment as C# - please update the method comments with the new `@param`.

Reply:
Done.